hax/hax/util.py

```diff
@@ -317,29 +322,31 @@ def get_session_node(self, session_id: str) -> str:
             raise HAConsistencyException('Failed to communicate to'
                                          ' Consul Agent') from e
 
+    def get_m0d_status(self, srv: ServiceData) -> Tuple[str, str]:
+        try:
+            key = f'processes/{srv.fid}'
+            raw_data = self.kv.kv_get(key)
+            LOG.debug('Raw value from KV: %s', raw_data)
+            data = raw_data['Value']
+            value: str = json.loads(data)['state']
```

Comment:
`json.loads(data)` is evaluated twice. No need to repeat it.

Reply:
Right, fixed.